unstructured/partition/pdf_image/pdfminer_processing.py

```diff
@@ -223,7 +223,7 @@ def clean_pdfminer_inner_elements(document: "DocumentLayout") -> "DocumentLayout
     """
 
     for page in document.pages:
-        table_boxes = [e.bbox for e in page.elements if e.type == ElementType.TABLE]
+        non_pdfminer_element_boxes = [e.bbox for e in page.elements if e.source != Source.PDFMINER]
```

Comment:
Is there any way to avoid adding the PDF miner elements to the list in the first place instead of filtering them after the fact? If that's complicated, definitely okay with moving forward with filtering for the bug fix and circling back on that later.

Reply:
I think we can move forward with filtering option right now and then circle back on this in a separate PR?